src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -3774,6 +3800,20 @@ inline void rpp_resize_nn_load_u8pln1(Rpp8u *srcRowPtrsForInterp, Rpp32s *loc, _
     p = _mm_unpacklo_epi8(px[0], px[1]);    // unpack to obtain [R01|R11|R21|R31|00|00|00|00|00|00|00|00|00|00|00|00]
 }
 
+inline void rpp_resize_nn_load_u8pln1_avx(Rpp8u *srcRowPtrsForInterp, Rpp32s *loc, __m256i &p)
+{
+    p = _mm256_setr_epi8(*(srcRowPtrsForInterp + loc[0]), *(srcRowPtrsForInterp + loc[1]), *(srcRowPtrsForInterp + loc[2]), *(srcRowPtrsForInterp + loc[2]),
+                         *(srcRowPtrsForInterp + loc[4]), *(srcRowPtrsForInterp + loc[5]), *(srcRowPtrsForInterp + loc[6]), *(srcRowPtrsForInterp + loc[7]),
+                         0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
+}
+
+inline void rpp_resize_nn_load_i8pln1_avx(Rpp8s *srcRowPtrsForInterp, Rpp32s *loc, __m256i &p)
```

Comment:
If this is the same for I8 as U8, pls just template them and change the name to rpp_resize_nn_load_pln1_avx().
And separate them with just 2 elements on each line instead of 4, for better readability.

Reply:
templating will throw errors, since we are using `rpp_simd_load `function to call these load function

Reply:
I am getting this error
` note: candidate template ignored: couldn't infer template argument 
'FuncType'                                                                  `

Reply:
Done